common/src/IO/MdlParser.cpp

```diff
@@ -296,7 +296,7 @@ namespace TrenchBroom {
             parseFrame(reader, model, frameIndex, surface, triangles, vertices, skinWidth, skinHeight, origin, scale);
         }
 
-        void MdlParser::parseSkins(Reader& reader, Assets::EntityModelSurface& surface, const size_t count, const size_t width, const size_t height, const int flags) {
+        void MdlParser::parseSkins(BufferedReader& reader, Assets::EntityModelSurface& surface, const size_t count, const size_t width, const size_t height, const int flags) {
```

Comment:
I'm not sure what this commit does, actually. It doesn't align with the description. I see what you changed here, but I don't understand why. Is there something missing in this commit?

Reply:
Nothing missing - I changed the Reader argument of `Palette::indexedToRgba` to a `BufferedReader` (not in this commit) and then adjusted all of the call sites so it would compile (what is in this commit)

Reply:
So I guess this commit does not compile, only the last one does?

Reply:
It compiles! BufferedReader is a subclass of Reader. If you run it, and set a breakpoint in `Palette::indexedToRgba`, you'll see it's always passed a BufferedReader in this commit. That's all this commit does

Reply:
Alright, thanks for clarifying.